lokilogs/lokilog.go

```diff
@@ -0,0 +1,86 @@
+package lokilogs
```

Comment:
should we be calling this `lokilogs` ? this does not have any direct relation to loki, this just enables access log and are written to stdout